### PR TITLE
Display detailed timestamp when hovering on age

### PR DESCRIPTION
**Pull request by csillag**
As suggested on slack.

### CODE REVIEW

**Comment by csillag** [comment]
Waiting for feedback from @donouwens. (Eesy to test behavior [here](https://d316d8b3.oasis-explorer.pages.dev/mainnet/emerald/block).) Just move the cursor above the ages.

**Comment by csillag** [comment]
The CI failure is due to an unrelated devops issue, we can ignore it.

File: src/app/pages/TransactionDetailPage/index.tsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ export const TransactionDetailView: FC<{
   const { t } = useTranslation()
   const { isMobile } = useScreenSize()
 
-  const formattedTimestamp = useFormattedTimestampString(transaction?.timestamp)
+  const formattedTimestamp = useFormattedTimestampStringWithDistance(transaction?.timestamp)
```
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/app/pages/TransactionDetailPage/index.tsx:182]: I would replace all usages of useFormattedTimestamp* with `<Age expanded`.
Expanded = isMobile ? "relative" + tooltip("full") : "full (relative)"

File: src/app/components/Age/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const date = new Date(sinceTimestamp)
+  const actualTime = useFormattedTimestamp(date)
+  const distance = formatDistanceToNow(date)
+  const distanceWithSuffix = formatDistanceStrict(sinceTimestamp, new Date(), {
+    addSuffix: true,
+  })
+  const title = (
+    <span>
+      {actualTime}
+      <Box fontWeight={22}>{distanceWithSuffix}</Box>
```
[the remark this answers is not in the document]
Comment (csillag) [reply src/app/components/Age/index.tsx:19]: It sets font weight to 22. Could achieve the same effect with `<span style="font-weight: 22">`...

File: src/app/pages/TransactionDetailPage/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (csillag) [reply src/app/pages/TransactionDetailPage/index.tsx:182]: > Maybe for separate PR - this doesn't use Age on mobile layout. It only shows relative time

We don't really have hovering on mobile, do we?

File: src/app/components/Age/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/app/components/Age/index.tsx:19]: yes, but why 22. I would expect 200, or 100.

And <100 even falls out of our `@font-face` definition

File: src/app/pages/TransactionDetailPage/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/app/pages/TransactionDetailPage/index.tsx:182]: Yes. Longpress

File: src/app/components/Age/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (csillag) [reply src/app/components/Age/index.tsx:19]: Indeed, that was a misunderstanding. Updated to 100.

File: src/app/pages/TransactionDetailPage/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (csillag) [reply src/app/pages/TransactionDetailPage/index.tsx:182]: >> We don't really have hovering on mobile, do we?

> Yes. Longpress

What?? For me, longpress on mobile is "right key", that is, bring up the context menu. (For example, on a link, open in new tab, open in new private tab, copy link, etc.) How do simulate hovering with longpress?

[the remark this answers is not in the document]
Comment (csillag) [reply src/app/pages/TransactionDetailPage/index.tsx:182]: > Maybe for separate PR - this doesn't use Age on mobile layout. It only shows relative time

Yes, let's do any further improvements in a separate PR.

[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/app/pages/TransactionDetailPage/index.tsx:182]: hm, then try longpress and slightly move the finger while holding :shrug:

**Pull request merged**